django_simple_forum/forms.py

```diff
@@ -88,14 +89,19 @@ def save(self, commit=True):
         return instance
 
 
+def cleanedSlug(title):
```

Comment:
@ravigadila advised to slugify the title with alphanumerics only. Even if the user enters any special characters, it'll slugify the title with alphanumerics only(Used regular-expressions to exclude rest of the characters).